OracleWebLogic/samples/12212-msiserver/container-scripts/launcher.sh

```diff
@@ -0,0 +1,19 @@
+#! /usr/bin/bash
```

Comment:
Is this behaviour replicated in the base WebLogic image or introduced by your addition of the `launcher.sh` script? If the former, please open another issue so that @mriccell can look into that. If the latter, you'll need to reimplement the handlers so that the `SIGTERM` is sent to the right `pid`.